cryptofeed/backends/elastic.py

```diff
@@ -0,0 +1,102 @@
+'''
+Copyright (C) 2017-2019  Bryant Moscon - bmoscon@gmail.com
+
+Please see the LICENSE file for the terms and conditions
+associated with this software.
+'''
+from decimal import Decimal
+import logging
+import json
+import itertools
+
+from cryptofeed.defines import BID, ASK
+from cryptofeed.backends._util import book_convert, book_delta_convert, book_flatten
+from cryptofeed.backends.http import HTTPCallback
+from cryptofeed.exceptions import UnsupportedType
+
+
+LOG = logging.getLogger('feedhandler')
+
+
+class ElasticCallback(HTTPCallback):
+    def __init__(self, addr: str, index=None, numeric_type=str, **kwargs):
+        super().__init__(addr, **kwargs)
+        self.addr = f"{addr}/{index}/{index}"
+        self.session = None
+        self.numeric_type = numeric_type
+
+
+class TradeElastic(ElasticCallback):
+    def __init__(self, *args, index='trades', **kwargs):
+        super().__init__(*args, index=index, **kwargs)
+
+    async def __call__(self, *, feed: str, pair: str, side: str, amount: Decimal, price: Decimal, order_id=None, timestamp=None):
```

Comment:
for this, it might be worth just letting it take the dict rather than individual fields, since the individual fields are the parameters that'll be in the trade dict anyway?

basically, for the use of this, do I need to break up a dict that has these same fields (if not then it's good as is).

Reply:
you can still use the ** to turn a dict into kwargs